docs/workforce-management-framework/topics/pro-working-with-passportjs.adoc

```diff
@@ -0,0 +1,41 @@
+[id='pro-working-with-passportjs-{chapter}']
+= Introduction
+
+This section includes:
+
+. xref:pro-configuring-passportjs-{chapter}[Configuring Passport.js]
+. xref:pro-implementing-the-userrepository-{chapter}[Implementing the UserRepository]
+. xref:pro-modifying-a-strategy-{chapter}[Modifying a Strategy]
+
+= Prerequisites
+(?)
```

Comment:
What are the they, if applicable?

Reply:
same as the getting started prerequisites so shouldn't apply them.